### ISSUE TITLE
Update board costs when build changes

### ISSUE THREAD

**Issue by trezy** [pcww/builder#8]

**Comment by voltaaage** [comment]
How is this calculated?

**Comment by brian-slate** [comment]
Groove/feet/edge/handles all change price by a fixed amount. Get these from jason, add to UI (see https://www.missionbicycle.com/bike-builder), and reflect in overall price.

In the meantime I think we should set a base price for each board. If you deviate from the base design (add/remove strips or change wood) you incur a "custom board" fee of like $75. 

We'll also need to talk to jason to figure out if we can do a rough calculation per strip as well so someone doesn't just customize a cheap board to look like a expensive board for cheaper hah. 

So bottom line: 
Default design: BASE_COST + ACCESSORY_COST
Custom Design: BASE_COST + ACCESSORY_COST + CUSTOM_BOARD_FEE + STRIPS_COST(?)
